chaosinstana/api.py

```diff
@@ -0,0 +1,66 @@
+# -*- coding: utf-8 -*-
+from typing import Any, Dict
+
+from chaoslib.types import Configuration, Secrets
+from logzero import logger
+import requests
+
+
+__all__ = ['call_api', 'get_all_events']
+
+
+def call_api(path: str, configuration: Configuration,
```

Comment:
I'm not sure I see the benefit of this function here considering its body.

Why not simply calling `r = requests.get(url)`  everywhere?

Is there a value for call_api?

Reply:
I think it's now removed @gtfisher ?